public/app/plugins/datasource/prometheus/querybuilder/parsing.test.ts

```diff
@@ -71,6 +71,70 @@ describe('buildVisualQueryFromString', () => {
     );
   });
 
+  // Visual query builder does not currently have support for nested binary operations, for now we should throw an error in the UI letting users know that their query will be misinterpreted
```

Comment:
Could you please put those tests in a `describe` section so it will be easy to spot when we need to work on them or change in the future?